packages/react-charting/docs/TestPlans/DonutChart/UnitTests.md

```diff
@@ -0,0 +1,16 @@
+# Unit test plan for Donut Chart
+
+This test plan contains the list of unit testable functions which are used as a part of the Donut Chart component.
+
+Identify the functions that can be unit tested (example, functions having calculations or getting values from Utils, etc).
+
+- If required, extract the unit testable portions out of the functions which can be independently unit tested without any requirement of DOM elements.
+- Alternatively, mock the sections that cannot be unit tested.
+
+| Functions                    | Can it be unit tested | Reason                                                                                                                                                                                 |
+| ---------------------------- | --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| a. convertToLocaleString()   | YES                   |                                                                                                                                                                                        |
+| b. getAccessibleDataObject() | YES                   |                                                                                                                                                                                        |
+| c. \_valueInsideDonut()      | NO                    | private function which can only be tested by rendering the donnut chart component.                                                                                                     |
+| d. wrapTextInsideDonut()     | NO                    | cannot be unit tested as it requires the tspan length to be calculated using Browser Functions like getComputedTextLength().                                                           |
```

Comment:
wrap all function names in backtick so that they are rendered as code by markdown.

Reply:
done